test/instance.ts

```diff
@@ -263,6 +256,23 @@ describe('Bigtable/Instance', () => {
         };
         instance.createAppProfile(APP_PROFILE_ID, options, assert.ifError);
       });
+
+      it('an array of clusters', done => {
```

Comment:
this this needs a description and change to "set"

Reply:
Changed to say `set`. Regarding description, the other tests in the same describe block say `'a cluster value'` and `"an 'any' value"`.  Should those descriptions change along with this to something like `should accept an 'any' value`?